admin/linux/build-appimage.sh

```diff
@@ -30,7 +30,7 @@ mkdir build
 cd build
 cmake -D CMAKE_INSTALL_PREFIX=/usr ../
 make -j4
-make DESTDIR=/app install 
```

Comment:
This looks unrelated?

Reply:
> This looks unrelated?

Well actually not, /app wasn't in the paths the cmake was looking at to find qtkeychain.